scripts/get-token.js

```diff
@@ -1,23 +1,59 @@
-import fs from 'fs'
-import dotenv from 'dotenv'
-import FirebaseTokenGetter from 'firebase-idtoken-getter'
+/* eslint-disable @typescript-eslint/no-var-requires */
+// const fs = require('fs')
+// const dotenv = require('dotenv')
+// const FirebaseTokenGetter = require('firebase-idtoken-getter')
+
+// const envConfig = dotenv.parse(fs.readFileSync('./.env'))
+
+// console.log(envConfig)
+
+// const FirebaseTokenGetterObject = new FirebaseTokenGetter(
+//   envConfig.GOOGLE_APPLICATION_CREDENTIALS,
+//   process.argv[2] ?? envConfig.FIREBASE_UID,
+//   envConfig.FIREBASE_API_KEY,
+//   envConfig.GOOGLE_CLOUD_PROJECT,
+// )
+
+// const getIdtoken = async () => {
+//   try {
+//     const token = await FirebaseTokenGetterObject.createIdTokenBycustomToken()
+//     console.log(token)
+//   } catch (error) {
+//     console.error(error)
+//     process.exit(1)
+//   }
+// }
+// getIdtoken()
+
+const fs = require('fs')
+const admin = require('firebase-admin')
+const dotenv = require('dotenv')
+const serviceAccount = require('../firebase.json')
```

Comment:
Location of this file should be located via `process.env.GOOGLE_APPLICATION_CREDENTIALS`.